snapcraft/_baseplugin.py

```diff
@@ -73,13 +73,21 @@ def PLUGIN_STAGE_SOURCES(self):
         """Define alternative sources.list."""
         return getattr(self, '_PLUGIN_STAGE_SOURCES', [])
 
+    @property
+    def stage_packages(self):
+        return self._stage_packages
+
+    @stage_packages.setter
+    def stage_packages(self, value):
+        self._stage_packages = value
```

Comment:
Using this property and setter combo, I don't see how this changes from using a simple property as we were doing. Am I missing something?